### ISSUE TITLE
How to make it work when starting touch on the child views?

### ISSUE THREAD

**Issue by eljago** [kiddkai/react-native-gestures#11]
I noticed that when starting the touch on a child view, the gestures doesn't work. 
In the example if I touch one of the Text views and try to drag, the GestureView won't move.

How could I make it ignore the child views?

**Comment by jadsonlourenco** [comment]
Yes, not work on child views, also why we need child as required prop?

**Comment by futurepaul** [comment]
I was having this problem as well and found this [helpful answer on stackoverflow](http://stackoverflow.com/questions/30041111/is-there-a-way-in-which-i-can-ignore-touch-events-on-text-in-react-native).

Basically you just say `pointerEvents="none"` on the thingy you want to ignore.

``` jsx
<GestureView
  ...(gestureview props)...
>
  <View pointerEvents="none">
    <Text>Heyyy</Text>
    <Text>Heyyy</Text>
  </View>
</GestureView>
```

**Comment by sscaff1** [comment]
How do I wrap the Gesture view around an image?

```
<GestureView ...gestureProps...>
  <Image source={{uri: uri}} />
</GestureView>
```

The above doesn't seem to work...Any ideas?

I created a pull request for the above. #13
